pages/pages.json

```diff
@@ -7,16 +7,19 @@
       "path": "/"
     },
     "store/account": {
-      "path": "/account"
+      "path": "/account",
+      "login": true
     },
     "store/account/orders": {
-      "path": "/account/orders"
+      "path": "/account/orders",
+      "login": true
     },
     "store/compare": {
       "path": "/compare"
     },
     "store/login": {
-      "path": "/login"
+      "path": "/login",
+      "loginPage": true
```

Comment:
I don't think we need this. We should, by convention, redirect users to `{root}/login` page.